src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -153,7 +153,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
     setChannelAsRead(channel);
     if (channel?.channelType === ANON_PM) {
       setSelectedChannelKey(1);
-      return openChat('SampleChatScreen');
+      return navigation.push('AnonymousChatScreen');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from navigating to 'SampleChatScreen' to 'AnonymousChatScreen' seems appropriate given the context of checking if the channel type is `ANON_PM`. However, please ensure that all necessary data and state are correctly passed and maintained when navigating to 'AnonymousChatScreen'. If any additional setup or teardown is required when switching between these screens, it should be handled appropriately.
```diff
-      return openChat('SampleChatScreen');
+      return navigation.push('AnonymousChatScreen');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->